### PR TITLE
fix html usage

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@scopatz ready for review!

File: regolith/builders/htmlbuilder.py [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 from regolith.builders.basebuilder import BuilderBase
+from regolith.fsclient import _id_key
 from regolith.sorters import ene_date_key, position_key
 from regolith.tools import (
     all_docs_from_collection,
     filter_publications,
     filter_projects,
     make_bibtex_file,
     document_by_value,
+    dereference_institution,
```
Comment (sbillinge) [inline regolith/builders/htmlbuilder.py]: personally I like this....

File: regolith/tools.py [hunk 2]
```diff
@@ -448,3 +451,27 @@ def number_suffix(number):
     else:
         suffix = {1: "st", 2: "nd", 3: "rd"}.get(number % 10, "th")
     return suffix
+
+
+def dereference_institution(input_record, gtx):
+    """Tool for dereferencing institutions"""
```
Comment (sbillinge) [inline regolith/tools.py]: the docstring is a bit terse.

I am not sure what arguments to use...I guess ``input_record`` is some ``presentations``  or ``papers`` collection or subcollection? Or what?  Nothing is returned, but ``input_record`` is updated in place?

"dereference" is a bit jargony.  OK in the function name, but what does it mean in the docstring?

File: regolith/tools.py [hunk 3]
```diff
@@ -453,12 +453,19 @@ def number_suffix(number):
     return suffix
 
 
-def dereference_institution(input_record, gtx):
-    """Tool for dereferencing institutions"""
+def dereference_institution(input_record, institutions):
+    """Tool for replacing placeholders for institutions with the actual
```
Comment (sbillinge) [inline regolith/tools.py]: thanks @CJ-Wright

File: news/editor_html.rst [hunk 4]
```diff
@@ -0,0 +1,20 @@
+**Added:**
+
+* Optional ``static_source`` key in the rc for the html build.k
```
Comment (scopatz) [inline news/editor_html.rst]: Is the `k` a typo?

**Pull request merged**